### ISSUE TITLE
Have you seen VS 2015 Roslyn tooltips?

### ISSUE THREAD

**Issue by dsaf** [MrJul/ReSharper.EnhancedTooltip#30]
It seems to be quite close to ET (R# 8.2.3) and while lacking important features has some interesting ones:

1) on constants they are also showing the constant's value;
2) they are omitting the full namespace on members;
3) extension methods are shown as "(extension)" not "(method)".
4) displaying info about operators (they are more like methods than keywords afterall);
5) "optional.Value.HasValue" is displayed as "double?.HasValue" not like "System.Nullable<double>.HasValue" - is this C#6 specific?

In my opinion all of the above would be a great addition to ET.

**Comment by MrJul** [comment]
1) ReSharper does too. ET does not: it should definitely be changed.
2) Not sure about this by default, could be an option IMO. Same thing with containing classes for nested classes, modifiers or constant value.
3) Easy and helpful change, will do.
4) Already done as a side effect of #17.
5) It's not C# 6 specific, VS2013 already displays `Nullable<T>.Value` as `T?.Value`. I find it weird as a member path, but that's completely subjective. ET mimics ReSharper here and doesn't use type keywords to refer to a member's type. Another option?

**Comment by dsaf** [comment]
2) option is good;
5) I don't know now :). In C#6 and its null propagation I might get slightly confused whether I am looking at a null-safe code handling a POCO that happens to have a Value property or an actual Nullable:

```csharp
public class FormulaEntry
{
   public double? Value { get; set; }
}
...
public void SafelyPrint(FormulaEntry entry)
{
   Console.WriteLine(entry?.Value.Value);
}
```

**Comment by MrJul** [comment]
1) and 3) fixed on f1aa2fe7cd0b9a92764bfad05d0306183731cbb3

**Comment by MrJul** [comment]
1, 3, 4 are fixed.
2 is #51
5 is #52
